src/hardware/scpi-pps/profiles.c

```diff
@@ -1222,6 +1373,46 @@ SR_PRIV const struct scpi_pps pps_profiles[] = {
 		.update_status = NULL,
 	},
 
+	/*
+	 * Envox EEZ PSU Series
+	 * The documented identification strings disagree with the behavior
+	 * of at least some real units (returning "EEZ"). The first of these
+	 * is the documented one, while the second seems to be returned by
+	 * firmware v1.02 and earlier.
+	 */
+	{ "Envox", "^EEZ H24005 ", SCPI_DIALECT_UNKNOWN, 0,
+		ARRAY_AND_SIZE(eez_psu_devopts),
+		ARRAY_AND_SIZE(eez_psu_devopts_cg),
+		NULL, 0,
+		NULL, 0,
+		eez_psu_cmd,
+		.probe_channels = eez_psu_probe_channels,
+		.init_acquisition = NULL,
+		.update_status = NULL,
+	},
+	{ "EEZ", "^PSU ", SCPI_DIALECT_UNKNOWN, 0,
```

Comment:
The string `PSU ` is a little bit blurry and could led to false positives when scanning devices.
Is it possible to make the regex unambiguous?

Reply:
Unfortunately this is the only part of the model name that is fixed... the firmware includes some dynamic information after this but not guaranteed to be the same for all configurations.

Reply:
Alternatively I could drop this particular profile altogether. It's here to support an early version of the PSU firmware, but the other pattern from the previous item is what the latest firmware uses, and that's been available for some time so perhaps reasonable to require firmware upgrade before using Sigrok.

Reply:
> I hoped that this prefix in conjunction with the manufacturer "EEZ" would be enough to distinguish it from other scpi-pps profiles.

Ah, I totally forgot about the check for the vendor string. With that in mind, this profile definition is fine!

Reply:
Great, thanks for confirming! I've left this in place in my updated commit, then.